test/extended/networking/network_segmentation.go

```diff
@@ -1919,7 +1919,6 @@ func generateNAD(config networkAttachmentConfig) *nadapi.NetworkAttachmentDefini
         "topology":%q,
         "subnets": %q,
         "excludeSubnets": %q,
-        "mtu": 1300,
```

Comment:
haha dejavu https://github.com/openshift/origin/commit/1a2cbe5cc45b330c682d582992b23a46e5a26ab3

Reply:
when you remove this what is the default value we set it to?